src/main/kotlin/com/nextstep/jngcii/racingcar/view/ResultView.kt

```diff
@@ -13,18 +13,19 @@ object ResultView {
         history.records
             .forEach { printSingleRecord(it) }
```

Comment:
요기서도 function reference를 활용해볼 수 있겠네요!
```suggestion
        history.records
            .forEach(::printSingleRecord)
```

Reply:
오 이렇게 사용하는 방법이 있었군요 🤩
오늘도 감사합니다!!